modules/src/ics24_host/path.rs

```diff
@@ -11,7 +11,7 @@ pub const IBC_QUERY_PATH: &str = "store/ibc/key";
 pub enum Path {
     ClientType(ClientId),
     ClientState(ClientId),
-    ClientConsensusState(ClientId, u64),
+    ClientConsensusState(ClientId, u64, u64),
```

Comment:
It may be easy to mix up the two `u64` fields, so let's perhaps give them names? What do you think?

```suggestion
    ClientConsensusState {
        client_id: ClientId,
        epoch: u64,
        height: u64,
    },
```

Reply:
Maybe we can also do the same for the other paths which contain a single, plain `u64`?